Automattic/social-logos
# Build: refactor build system to align to Gridicons'

This refactoring of the whole build system makes the repository aligned with the same build system of Gridicons.

Benefits:

* SVG are masters
* More modular, we can share tasks across
* Cleanly separated sources folder
* More automation
* Less dependencies (font-forge not required)

---

To test: 

* Try to run the build from scratch: `npm install` + `npm run build`.
* Check all the built files: React component, Demo, SVGs, PDFs, font file.
* Make sure there's no regression from master.

---

These are the corresponding Gridicons/Dashicons main changes to the build system for reference:

* https://github.com/Automattic/gridicons/pull/241
* https://github.com/Automattic/gridicons/pull/230
* https://github.com/Automattic/gridicons/pull/217
* https://github.com/WordPress/dashicons/pull/215
